biogeochem/EDPatchDynamicsMod.F90

```diff
@@ -994,19 +1050,31 @@ subroutine fire_litter_fluxes(currentSite, cp_target, new_patch_target, patch_si
        currentCohort => new_patch%shortest
        do while(associated(currentCohort))
 
+          sapw_c   = currentCohort%prt%GetState(sapw_organ, all_carbon_species)
+          leaf_c   = currentCohort%prt%GetState(leaf_organ, all_carbon_species)
+
           call carea_allom(currentCohort%dbh,currentCohort%n,currentSite%spread,currentCohort%pft,currentCohort%c_area)
+
           if(EDPftvarcon_inst%woody(currentCohort%pft) == 1)then
-             burned_leaves = currentCohort%bl * currentCohort%fraction_crown_burned
+             burned_leaves = leaf_c * currentCohort%fraction_crown_burned
```

Comment:
So here leaf_c is a mass and this X area fraction gives remaining mass of burned leaves?

Reply:
It's not an area fraction, it's more like a fraction of the crown burned, calculated from flame height.

Reply:
so a 1D fraction so say, 5kg of leaf biomass * 40% burned crown = 2 kg of living biomass / 3 kg of burned?  where burned_frac = f(flame height)?

Reply:
Yes, that sort of thing.